tools/sort.py

```diff
@@ -368,7 +370,8 @@ def calc_landmarks_face_yaw(fl):
             l = ( (fl[27][0]-fl[0][0]) + (fl[28][0]-fl[1][0]) + (fl[29][0]-fl[2][0]) ) / 3.0   
             r = ( (fl[16][0]-fl[27][0]) + (fl[15][0]-fl[28][0]) + (fl[14][0]-fl[29][0]) ) / 3.0
             return r-l
-            
```

Comment:
I can't see that face_recognition is used in this method. Just FaceLandmarksExtractor